compiler/packages/babel-plugin-react-compiler/src/HIR/Globals.ts

```diff
@@ -455,11 +466,15 @@ for (const [name, type_] of TYPED_GLOBALS) {
   DEFAULT_GLOBALS.set(name, type_);
 }
 
-// Recursive global type
+// Recursive global types
 DEFAULT_GLOBALS.set(
   'globalThis',
   addObject(DEFAULT_SHAPES, 'globalThis', TYPED_GLOBALS),
 );
+DEFAULT_GLOBALS.set(
+  'global',
```

Comment:
`window` too.

Reply:
I debated on adding window but figured we should leave it out for now. I wonder if we want to handle `window` and `document` separately as people often use these for DOM-manipulation (global mutable state)